### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by amanoese**
textare vertical is middle
and prev request mixed in tab is delete

### CODE REVIEW

File: src/components/Sticky.vue [hunk 1]
```diff
@@ -82,8 +86,8 @@ export default {
     mouseDown(e) {
       e.cancelBubble = true
       this.dragging = true
-	  this.layerPos.x = this.$el.offsetWidth / 2 - e.pageX + this.$el.offsetLeft
-	  this.layerPos.y = this.$el.offsetHeight / 2 - e.pageY + this.$el.offsetTop
+      this.layerPos.x = this.$el.offsetWidth / 2 - e.pageX + this.$el.offsetLeft
+      this.layerPos.y = this.$el.offsetHeight / 2 - e.pageY + this.$el.offsetTop
```
Comment (lunardog) [inline src/components/Sticky.vue:90]: please change to spaces

File: src/components/Sticky.vue [hunk 2]
```diff
@@ -9,9 +9,13 @@
       v-on:mouseleave="mouseLeave"
       v-on:mouseleave="mouseUp"
       v-on:mousemove="mouseMove">
-    <textarea
-      v-on:keyup="updateText"
-      v-model="sticky.text"></textarea>
+    <div class="position">
+      <div
+        class="textarea"
+        v-bind:contenteditable="true"
```
Comment (lunardog) [inline src/components/Sticky.vue]: how about just `contenteditable="true"`, no `v-bind`?

File: src/components/Sticky.vue [hunk 3]
```diff
@@ -190,7 +198,7 @@ export default {
 }
 
 
-textarea:focus, input:focus{
+.textarea:focus, input:focus{
   outline: none;
```
Comment (lunardog) [inline src/components/Sticky.vue:200]: can delete this section now.

File: src/components/Sticky.vue [hunk 4]
```diff
@@ -154,13 +158,17 @@ export default {
...
   cursor: move;
   left: 0;
   top: 0;
   width: 80%;
   height: 100%;
+}
+.sticky .textarea {
   text-align: center;
   border: none;
   background: none;
```
Comment (lunardog) [inline src/components/Sticky.vue]: can delete `border: none; background: none;`, default for `<div>`

**Comment by lunardog** [comment]
Please pull latest master into your branch to avoid conflicts.

File: src/components/Sticky.vue (hunk 3 above)
Comment (amanoese) [inline src/components/Sticky.vue:200]: but div has outline when it is contenteditable
this line is need

File: src/components/Sticky.vue (hunk 1 above)
Comment (amanoese) [inline src/components/Sticky.vue:90]: sorry, yesterday pull request included tab that was merged master.
this commit is replace tab to space

**Comment by lunardog** [comment]
OK to merge, after removing conflicts (pull master into this branch)

**Comment by lunardog** [comment]
OK, I merged.

**Pull request merged**